tools/buildsys/src/manifest.rs

```diff
@@ -152,6 +162,15 @@ pub(crate) struct BuildPackage {
 #[serde(rename_all = "kebab-case")]
 pub(crate) struct BuildVariant {
     pub(crate) included_packages: Option<Vec<String>>,
+    pub(crate) image_format: Option<ImageFormat>,
+    pub(crate) image_options: Option<Vec<String>>,
+}
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "lowercase")]
+pub(crate) enum ImageFormat {
+    IMG,
```

Comment:
Can we refer to `IMG` instead as `Raw`? The sort of files that `IMG` is naming are typically called `raw` by tools and the various virtualization communities.